packages/fast-component-explorer/app/utilities/configs/lightweight-button.ts

```diff
@@ -0,0 +1,62 @@
+import { ComponentViewConfig } from "./data.props";
+import {
+    LightweightButton,
+    LightweightButtonProps,
+    lightweightButtonSchema,
+} from "@microsoft/fast-components-react-msft";
+import Guidance from "../../.tmp/accent-button/guidance";
+import { glyphSchema, Icon } from "../../../app/components/glyph";
+
+const lightweightButtonConfig: ComponentViewConfig<LightweightButtonProps> = {
+    schema: lightweightButtonSchema,
+    component: LightweightButton,
+    guidance: Guidance,
+    scenarios: [
+        {
+            displayName: "Default",
+            data: {
+                children: "Lightweight button",
+            },
+        },
+        {
+            displayName: "Anchor",
+            data: {
+                href: "#",
+                children: "Anchor lightweight button",
+            },
+        },
+        {
+            displayName: "Disabled",
+            data: {
+                disabled: true,
+                children: "Disabled lightweight button",
+            },
+        },
+        {
+            displayName: "Before content lightweight button",
```

Comment:
This doesn't read well, maybe "Glyph before text"

Reply:
I agree it reads strange, but `beforeContent` and `afterContent` could potentially be something other than a glyph. How about "Content before text"?

Reply:
Discussed offline and updated.